microprofile/5.0/config/3.0/22.0.0.1-TCKResults-java11.adoc

```diff
@@ -0,0 +1,468 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Config 3.0.1.
+
+== Open Liberty 22.0.0.1 - MicroProfile Config 3.0.1 Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/22.0.0.1/openliberty-runtime-22.0.0.1.zip[Open Liberty 22.0.0.1]
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-config-3.0.1/microprofile-config-3.0.1.html[MicroProfile Config 3.0.1]
+
+* Public URL of TCK Results Summary:
++
+link:22.0.0.1-TCKResults.html[TCK results summary]
```

Comment:
This link to self isn't right now. I wonder if just using a dot would resolve correctly?? Just a guess, not tested
```suggestion
link:.[TCK results summary]
```